viper/src/main/java/viper/api/VerCorsExpressionFactory.java

```diff
@@ -76,6 +76,16 @@ public ASTNode any_set_intersection(Origin o, ASTNode e1, ASTNode e2) {
     return res;
   }
 
+  @Override
+  public ASTNode any_set_subset(Origin o, ASTNode e1, ASTNode e2) {
+      enter(o);
+      //TODO check what to put here.
```

Comment:
I believe this is OK since at least Java and C will not have a sets or bags as basic types? thus they do not have this operation? or am I wrong. If not, then we may remove the comment.

Reply:
Good that you found the TODO, that should have been removed as the comment is not applicable anymore. Will be removed in the next commit.